daprdocs/content/en/operations/support/support-release-policy.md

```diff
@@ -101,6 +101,7 @@ After announcing a future breaking change, the change will happen in 2 releases
 |-----------------------|-----------------------|------------------------- |
 | GET /v1.0/shutdown API (Users should use [POST API]({{< ref kubernetes-job.md >}}) instead)             |                 1.2.0 |                    1.4.0 |
 | Java domain builder classes deprecated (Users should use [setters](https://github.com/dapr/java-sdk/issues/587) instead) | Java SDK 1.3.0 | Java SDK 1.5.0 |
+| Service invocation will no longer provide a default content type header of `application/json` when no content-type is specified. You must explictly set a content-type header for service invocation if your invoked apps rely on this header. |                 1.7.0 |                    1.8.0 |
```

Comment:
Also the policy for breaking changes and deprecation is 2 releases. We should say this is default in the v1.9 release.